setup.py

```diff
@@ -43,7 +43,7 @@ def run(self):
     version="2.0.0",
     packages=find_packages(),
     install_requires=[
-        'django==1.8',
+        'django>=1.8',
```

Comment:
You might want to use something like `django>=1.8,<1.9` instead, so things don't explode when 1.9 comes out of beta.